src/main/webapp/resources/js/pages/analysis/components/AnalysisDetails.jsx

```diff
@@ -126,43 +126,39 @@ export function AnalysisDetails() {
   // The following renders the Analysis Details component view
   return (
     <>
-        <Title level={2}>{getI18N("AnalysisDetails.details")}</Title>
-
-        <div>
-          <List
-            itemLayout="horizontal"
-            dataSource={analysisDetails}
-            renderItem={item => (
-              <List.Item>
-                <List.Item.Meta
-                  title={
-                    <span style={{ fontWeight: "bold" }}>{item.title}</span>
-                  }
-                  description={<span>{item.desc}</span>}
-                />
-              </List.Item>
-            )}
-          />
-        </div>
-
-        {window.PAGE.mailConfigured &&
-        !analysisContext.isCompleted &&
-        !analysisContext.isError ? (
-          <section
-            style={{ marginTop: SPACE_MD }}
-            className="t-email-pipeline-result"
+      <Title level={2}>{getI18N("AnalysisDetails.details")}</Title>
+
+      <div>
+        <List
+          itemLayout="horizontal"
+          dataSource={analysisDetails}
+          renderItem={item => (
+            <List.Item>
+              <List.Item.Meta
+                title={<span style={{ fontWeight: "bold" }}>{item.title}</span>}
```

Comment:
Consider using the `Typography` element for this.

Reply:
Will do

Reply:
Fixed in dc1a014b789d7270dd33c9e420a21a122a5e796a